sync/bug.py

```diff
@@ -286,3 +308,9 @@ def __setitem__(self, name, value):
     def needinfo(self, *requestees):
         for requestee in requestees:
             self.changes.append("Setting bug %s needinfo %s" % (self.bug_id, requestee))
+
+    def add_comment(self, text, check_dupe=True):
+        self.comment = text
```

Comment:
I'm curious, if you include the "one comment per bug" ValueError here, do the tests still pass?